src/web/app/pages-session/session-result-page/session-result-page.component.ts

```diff
@@ -150,11 +150,10 @@ export class SessionResultPageComponent implements OnInit {
           feedbackSessionName: this.feedbackSessionName,
           studentEmail: this.personEmail,
           logType: LogType.FEEDBACK_SESSION_VIEW,
-        }).subscribe(() => {
-
-        }, () => {
-          this.statusMessageService.showWarningToast('Failed to log feedback session view');
-        });
+        }).subscribe(
+          // No action needed if log is successfully created
+          () => {},
```

Comment:
Thanks. This reads better to me despite existing pattern elsewhere.
Do you want to do the following?
`() => { /* No action needed if log is successfully created. */ },`

Reply:
Yep. This looks better